### PR TITLE
Adapt to Android Gradle Plugin 3.3.1

### PR DESCRIPTION
**Pull request by mattiasrosberg**
This will get rid of the warnings discussed in #167

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -17,6 +17,7 @@ before_install:
   - sudo hostname "$(hostname | cut -c1-63)"
   - sed -e "s/^\\(127\\.0\\.0\\.1.*\\)/\\1 $(hostname | cut -c1-63)/" /etc/hosts | sudo tee /etc/hosts
   - cat /etc/hosts # optionally check the content *after*
+  - yes | sdkmanager "platforms;android-26"
```
Comment (autonomousapps) [inline .travis.yml]: did you mean this to be `28`?

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -1,5 +1,6 @@
+#Mon Feb 11 09:32:01 CET 2019
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.3.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.1-all.zip
```
Comment (autonomousapps) [inline gradle/wrapper/gradle-wrapper.properties]: 4.10.3 and 5.2.1 also exist.

File: .travis.yml (hunk 1 above)
Comment (mattiasrosberg) [reply .travis.yml]: I tried that initially but that didn't work so I switched back to the 26. I found this post indicating that there is some bug related to the acceptance of the 28 licence. https://travis-ci.community/t/accept-license-for-android-build-tools-27-0-3-bug/549

File: gradle/wrapper/gradle-wrapper.properties (hunk 2 above)
Comment (mattiasrosberg) [reply gradle/wrapper/gradle-wrapper.properties]: I have updated to 4.10.3

File: .travis.yml (hunk 1 above)
Comment (AndrewReitz) [reply .travis.yml]: I think you can do this `printf "d56f5187479451eabf01fb78af6dfcb131a6481e\n24333f8a63b6825ea9c5514f83c2829b004d1fee" > android-sdk-linux/licenses/android-sdk-license` as a work around

**Comment by AndrewReitz** [comment]
Thanks for the PR!

**Pull request merged**